holoviews/util/__init__.py

```diff
@@ -339,6 +339,78 @@ def __call__(self, *args, **params):
         Store.current_backend = selected_backend
 
 
+def save(obj, filename, fmt='auto', backend=None, **kwargs):
+    """
+    Saves the supplied object to file.
+
+    The available output formats available depend on the backend being
+    used. By default and if the filename is a string the output format
+    will be inferred from the file extension. Otherwise an explicit
+    format will be specified. For ambiguous file extensions such as
+    html it may be necessary to specify an explicit fmt to override
+    the default.
```

Comment:
That last bit is confusing to me -- what fmt other than 'html' would be more explicit?  If you mean widgets vs. scrubber, please say that explicitly -- i.e. that `.html` will by default be XX, but if you don't want that, specify YY.

Reply:
Sounds good.